.github/workflows/build.yaml

```diff
@@ -0,0 +1,84 @@
+name: Build
+on: [workflow_dispatch, pull_request]
+env:
+  TERM: xterm-256color
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - name: Build - Ubuntu
+            runs-on: ubuntu-latest
+
+          - name: Build - Windows (win32)
+            runs-on: windows-latest
+            cmake-args: -A Win32
+
+          - name: Build - Windows (win64)
+            runs-on: windows-latest
+            cmake-args: -D CMAKE_GENERATOR_PLATFORM=x64
```

Comment:
This should be default and `if(NOT CMAKE_GENERATOR_PLATFORM STREQUAL "Win32"` should be used in cmake instead of `if(CMAKE_GENERATOR_PLATFORM STREQUAL "x64"`

Reply:
There is another PR that updates Cmake.

Reply:
Removed `-D CMAKE_GENERATOR_PLATFORM=x64`.